src/components/UI/Card/CartCards/CartCards.jsx

```diff
@@ -1,88 +1,137 @@
                 <div className="cart-card-horizontal-layout">
                     <div className="card-content">
                         <div className="card-body">
-                            {title}
-                        </div>
+                            {title} </div>
                         <div className="card-body">
                             <h2>₹{price}</h2>
-                    <span className="text-grey">{discount}</span>
-                        </div>   
-                        <div className="card-body">
-                            Quantity
-                            <select>
-                                <option>1</option>
-                                <option>2</option>
-                                <option>3</option>
-                                <option>4</option>
-                            </select>
+                            <span className="text-grey">
+                                {discount}</span>
                         </div>
-                    </div> 
+                        <div className="card-footer-view">
+                        <button className='btn-addToCart'>
+                            {
+                           <div  className='cart-quantity-container'>
+                                <span onClick={
+                                    () => UpdateCartHandler("increment")
+                                }><IcRoundPlus/></span>
+                                {
+                                {...cartState.find((cartitem) => cartitem._id === _id)}?.qty}
+                                <span onClick={
+                                    () => UpdateCartHandler("decrement")
+                                }><IcRoundMinus/></span>
+                            </div> 
+                        } </button>
+                    </div>
+                    </div>
                     <div className="card-footer-view cart-card-footer cart-card-footer-flex">
-                        <button onClick={()=>{RemoveItemsFromCartHandler(_id)}}>Remove from Cart<span className="material-icons-round">
-                            shopping_cart
-                            </span>
+                        <button onClick={
+                            () => {
+                                RemoveItemsFromCartHandler(_id)
+                            }
+                        }>Remove from Cart
+                            <IcTwotoneShoppingCartCheckout/>
+                            {/* <span className="material-icons-round">
+                                shopping_cart
+                            </span> */}
 
-                            </button>
-                        <button onClick={()=>{addToWishlistHandler(props)}}>Move to Wishlist</button>
+                        </button>
+                        <button onClick={
+                            () => {
+                                addToWishlistHandler(props)
+                            }
```

Comment:
You could have written a single-line arrow anonymous function.

Reply:
actually I did that only the prettier vscode extension does it